src/udp_wrap.cc

```diff
@@ -324,6 +328,43 @@ void UDPWrap::DropMembership(const FunctionCallbackInfo<Value>& args) {
   SetMembership(args, UV_LEAVE_GROUP);
 }
 
+void UDPWrap::SetSourceMembership(const FunctionCallbackInfo<Value>& args,
+                            uv_membership membership) {
```

Comment:
Please line `uv_membership` up vertically with `const` from the previous line.

Reply:
fixed in 305693a